src/applications/check-in/components/pages/validate/ValidateDisplay.unit.spec.jsx

```diff
@@ -70,16 +139,68 @@ describe('check-in experience', () => {
       describe('last4Input', () => {
         it('displays the value', () => {
           const { getByTestId } = render(
-            <ValidateDisplay last4Input={{ last4Ssn: 'foo' }} />,
+            <Provider store={store}>
+              <I18nextProvider i18n={i18n}>
+                <ValidateDisplay last4Input={{ last4Ssn: 'foo' }} />
+              </I18nextProvider>
+            </Provider>,
           );
 
           expect(getByTestId('last-4-input').value).to.equal('foo');
         });
       });
+      describe('dobInput', () => {
+        it('displays the value', () => {
+          const updatedStore = {
+            checkInData: {
+              context: {
+                token: '',
+              },
+            },
+            featureToggles: {
+              check_in_experience_lorota_security_updates_enabled: true,
+            },
+          };
+          const { getByTestId } = render(
+            <Provider store={mockStore(updatedStore)}>
+              <I18nextProvider i18n={i18n}>
+                <ValidateDisplay
+                  dobInput={{
+                    dob: {
+                      day: {
+                        value: 1,
+                        dirty: false,
```

Comment:
This is me being too lazy to look at the dob component but do we need any tests around what happens if `dirty` is true? Also I'd like to see a case with an ambiguous date like 3/12.

Reply:
Will add a test in e2e for error state.